pinning-services/index.js

```diff
@@ -199,6 +249,21 @@ async function getDatSeed(datSeedName, projName, domain, recordName) {
   }
 }
 
+function addDomainAccountDigitalOcean(domain, doToken) {
+  const url = `https://api.digitalocean.com/v2/domains`;
+  const headers = {
+    'Content-Type': 'application/json',
+    'Authorization': `Bearer ${doToken}`
+  };
+  const data = JSON.stringify({
+    name: domain
+  });
+  console.log(`POST ${url} with data ${data}`);
+  return fetch(url, { method: 'POST', headers: headers, body: data })
+    .then(res => res.json())
+    .then(json => json['domain'] != undefined);
+}
+
 function updateDnsRecordDigitalOcean(domain, recordType, recordName, recordData, recordTtl, doToken) {
```

Comment:
This is specific enough behaviour that you'll want to note it.

```suggestion
// Updates a DNS record. Passing '' as recordData deletes the record
function updateDnsRecordDigitalOcean(domain, recordType, recordName, recordData, recordTtl, doToken) {
```